README.md

```diff
@@ -1,2 +1,60 @@
 # grafink
-Repository hosting GSoC 2020 development on graphs
+Grafink is a spark ETL job to load data into Janusgraph.
+
+### Compiling from source
+
+```
+sbt compile
+```
+
+To compile against scala 2.11
+
+```
+sbt ++2.11.11 compile
+```
+
+### Code format
+
+This project uses scalafmt to format code. For formatting code:
+
+```
+sbt scalafmt        // format sources
+sbt test:scalafmt   // format test sources
+sbt sbt:scalafmt    // format.sbt source
+```
+
+### Running unit tests
+
+```
+sbt test
+```
+
+### Creating Assembly jar
+
+```
+sbt assembly
+```
+
+### Creating Distribution
+
+```
+sbt dist
+```
+
+The above creates a deployable zip file `grafink-<version>.zip`. The contents of the zip file are:
+
+  - conf/application.conf  // Modify this config file according to the job requirements.
+  - grafink assembly jar   // The main executable jar for running spark job.
+  - bin/start.sh           // The main executable script that user can invoke to start the job.
+
+For compiling and packaging against scala 2.11:
+
+```
+sbt ++2.11.11 dist
+```
+
+### Running Job
+
+```
+./bin/start.sh --config conf/application.conf
+```
```

Comment:
If I test the script locally with `SPARK_MASTER=local[*]`, i get:

```
grafink-0.1.0-SNAPSHOT julien$ ./bin/start.sh --config "conf/application.conf"
Running spark-submit...
20/05/26 08:24:41 WARN SparkConf: The configuration key 'spark.yarn.executor.memoryOverhead' has been deprecated as of Spark 2.3 and may be removed in the future. Please use the new key 'spark.executor.memoryOverhead' instead.
20/05/26 08:24:41 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
Error: Option --config must be a valid file path
janusloader 0.13.0
Usage: janusloader [options]

  -c, --config <value>  config accepts path to a configuration file
log4j:WARN No appenders could be found for logger (org.apache.spark.util.ShutdownHookManager).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```

what do I miss?